OfficeApp3Web/Scripts/InteractiveSDKv2.js

```diff
@@ -466,16 +467,29 @@ InteractiveTutorial.App = new function () {
 
             var run = $("<button id='run' class='buttonclass' accesskey='R'><u>R</u>un Code</button>").appendTo(navigation).click(self.executeCode);
 
-            //Loading last task with no Napa, show list button. Last step with Napa after, show next button 
-            if (_currentTaskIndex == _tasks.length - 1 && !hasNapaLink) {
-                //show Next button as Tutorial List icon
-                $("<div class='navigationButtons'><div id='previous' role='button' title='Go to the previous step'></div><div id='next' role='button' tabindex='0' title='Go to tutorial list'><img id='imgNext_WhiteBackground' src='Images/list-translucent.png' alt='Go to tutorial list' /><img id='imgNext_BlackBackground' src='/Images/list-translucent-highContrast.png' alt='Go to tutorial list' /></div></div>").appendTo(navigation);
-                _checked[_currentScenario] = true;
-            } else {
-                //show Next button as Next icon
-                $("<div class='navigationButtons'><div id='previous' role='button' title='Go to the previous step'></div><div id='next' role='button' tabindex='0' title='Go to the next step'><img id='imgNext_WhiteBackground' src='Images/next-translucent.png' alt='Next' /><img id='imgNext_BlackBackground' src='/Images/next-translucent-highContrast.png' alt='Next' /></div></div>").appendTo(navigation);
+            if (browser == "Netscape") {
```

Comment:
change logic of using both but one as hidden in html, then use css to hide/show for high contrast mode